### ISSUE TITLE
Bug: installation from latest install.sh would fail after #2235

### ISSUE THREAD

**Issue by Yiyiyimu** [chaos-mesh/chaos-mesh#2323]
## Bug Report

**What version of Kubernetes are you using?**
1.21.4

**What version of Chaos Mesh are you using?**
lastest

**What did you do?**
Install chaos mesh with latest `script.sh` in CI, which results in 
```
runner@fv-az204-48:~/work/apisix/apisix$ kubectl get pods -n chaos-testing
NAME                                        READY   STATUS             RESTARTS   AGE
chaos-controller-manager-766fc785f9-d5bm2   0/1     CrashLoopBackOff   9          23m
chaos-daemon-p928g                          1/1     Running            0          23m
chaos-dashboard-56d8f8cc7-g8jt8             0/1     CrashLoopBackOff   9          23m
```
The failure started from 4 hours ago (around 20:00 GMT+8), so I guess it might relate to #2235.

And here is the log:

[cm-error.log](https://github.com/chaos-mesh/chaos-mesh/files/7187004/cm-error.log)
[dashboard-error.log](https://github.com/chaos-mesh/chaos-mesh/files/7187005/dashboard-error.log)

**Comment by STRRL** [comment]
I noticed that in controller-manager's log:
```
 ERROR   controller-runtime.source       if kind is a CRD, it should be installed before calling Start   {"kind": "PhysicalMachineChaos.chaos-mesh.org", "error": "no matches for kind \"PhysicalMachineChaos\" in version \"chaos-mesh.org/v1alpha1\""}
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1
        /tmp/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.5/pkg/source/source.go:127
2021-09-18T03:49:01.157Z        ERROR   controller-runtime.manager.controller.physicalmachinechaos-condition    Could not wait for Cache to sync        {"reconciler group": "chaos-mesh.org", "reconciler kind": "PhysicalMachineChaos", "error": "failed to wait for physicalmachinechaos-condition caches to sync: no matches for kind \"PhysicalMachineChaos\" in version \"chaos-mesh.org/v1alpha1\""}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2
        /tmp/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.5/pkg/internal/controller/controller.go:195
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start
        /tmp/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.5/pkg/internal/controller/controller.go:221
sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1
        /tmp/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.5/pkg/manager/internal.go:696
```

We introduced a new CRD `PhysicalMachineChaos`, and you need to manually update the CRD.

**Comment by Yiyiyimu** [comment]
Thanks for the reply @STRRL!

Actually I install chaos mesh in a brand new environment (in CI) so I think CRD is directly installed with `install.sh` and the problem might not relate to the update.

CI script: https://github.com/apache/apisix/blob/release/2.9/.github/workflows/chaos.yml#L73

**Comment by STRRL** [comment]
tldr; please retry it, I think it should work now.

`install.sh` use CDN to deliver the `crd.yaml` files, maybe it did not update at first. Anyway, `install.sh` works fine on my machine now. :)

**Comment by Yiyiyimu** [comment]
It also works on my side. Thank you @STRRL!
